src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -1445,18 +1434,7 @@ export default {
 			this.loanDetailsVisible = true;
 		}
 	},
-	beforeRouteEnter(to, from, next) {
-		next(vm => {
-			if (vm.$refs.loandisplayref) {
-				vm.$refs.loandisplayref?.updateFromParams(to.query);
```

Comment:
This was also meant to power paging in grid view. specifically the ability to hit page=2 and load those loans...

Reply:
yeah this appears to work just fine...  did you notice an issue with it?

Reply:
reload the page and it doesn't work for me